tiledb/sm/c_api/tiledb_serialization.h

```diff
@@ -153,6 +153,48 @@ TILEDB_EXPORT int32_t tiledb_deserialize_array_schema(
     int32_t client_side,
     tiledb_array_schema_t** array_schema) TILEDB_NOEXCEPT;
 
+/**
+ * Serializes the given array open information into the given buffer.
+ *
+ * @note The caller must free the returned `tiledb_buffer_t`.
+ *
+ * @param ctx The TileDB context.
+ * @param array The array to get the information to serialize from.
+ * @param serialization_type Type of serialization to use.
+ * @param client_side If set to 1, deserialize from "client-side" perspective.
```

Comment:
ah here we are ...

... but i still don't know what this _means_ -- somewhere we need a one-sentence something about what a 'client-side perspective' is ...